puppet-mode.el

```diff
@@ -987,7 +987,7 @@ Used as `syntax-propertize-function' in Puppet Mode."
 ;;; Dealing with strings
 (defun puppet-looking-around (back at)
   "Check if looking backwards at BACK and forward at AT."
-  (and (looking-at-p at) (looking-back back)))
+  (and (looking-at-p at) (looking-back back nil)))
```

Comment:
It has two required parameters when taking the advertised calling convention into account, i.e. it's meant to have two required parameters, this is just not enforced by the interpreter (but the byte compiler warns nevertheless).